site/pages/create-disruption/[disruptionId].page.tsx

```diff
@@ -392,32 +392,48 @@ const CreateDisruption = (props: DisruptionPageProps): ReactElement => {
                                 <DateSelector<DisruptionInfo>
                                     display="Publication start date"
                                     hint={{ hidden: false, text: "Enter in format DD/MM/YYYY" }}
-                                    value={pageState.inputs.publishStartDate}
+                                    value={
+                                        pageState.inputs.publishStartDate ||
+                                        validity.disruptionRepeats !== "doesntRepeat"
+                                            ? pageState.inputs.publishStartDate
+                                            : validity.disruptionStartDate
+                                    }
                                     disabled={false}
                                     disablePast={false}
                                     inputName="publishStartDate"
                                     stateUpdater={stateUpdater}
                                     initialErrors={pageState.errors}
                                     schema={createDisruptionSchema.shape.publishStartDate}
                                     resetError={
+                                        (!pageState.inputs.publishStartDate &&
+                                            !!validity.disruptionStartDate &&
+                                            validity.disruptionRepeats === "doesntRepeat") ||
                                         pageState.inputs.publishStartDate ===
-                                        convertDateTimeToFormat(new Date(), "DD/MM/YYYY")
+                                            convertDateTimeToFormat(new Date(), "DD/MM/YYYY")
```

Comment:
end time and publish time is not mirroring correctly when i change 1100 to 1107 it isn't reflected